editor/src/clj/editor/scene.clj

```diff
@@ -1020,13 +1037,17 @@
                           (cond-> (.y max-p) zero-y inc)
                           (cond-> (.z max-p) zero-z inc)]))))
 
-(defn frame-selection [view animate?]
-  (let [aabb (fudge-empty-aabb (g/node-value view :selected-aabb))
-        camera (view->camera view)
-        viewport (g/node-value view :viewport)
-        local-cam (g/node-value camera :local-camera)
-        end-camera (c/camera-frame-aabb local-cam viewport aabb)]
-    (set-camera! camera local-cam end-camera animate?)))
+(defn frame-selection
+  ([view animate?]
+   (let [aabb (fudge-empty-aabb (g/node-value view :selected-aabb))]
+     (frame-selection view animate? aabb)))
```

Comment:
I already have AABB and if I try to get `selected-aabb` in my case, It might endup in an infinite loop (in case both `:scene-aabb and :selected-aabb` evicted from cache)